src/components/Trade/TradeInput.tsx

```diff
@@ -176,6 +210,9 @@ export const TradeInput = () => {
     selectPortfolioCryptoPrecisionBalanceByFilter(state, sellAssetBalanceFilter),
   )
 
+  const doesReceiveChainSupportYat = buyAsset.chainId === ethChainId // yat only supports eth mainnet
```

Comment:
Very semantnitpick but, `isYatSupportedByReceiveChain` or `receiveChainSupportsYat`?

Reply:
A much better name 😀 Addressed in https://github.com/shapeshift/web/pull/4479/commits/ae7253c23f0352f585bab637223ea053ef5e3843.